user-guide/source/tutorial/07-plugins.rst

```diff
@@ -137,6 +137,13 @@ URL - will be automatically kept up-to-date.
 .. image:: ./images/07-text-enabled-plugins.jpg
     :alt: The CMS Plugins menu in the rich text editor
 
+.. tip::
+
+    To remove "CMS Plugins" menu within the text editor, click on the editor and hit
+    the backspace or delete key to remove it.
+
+    For "Icon" CMS Plugin, click on "Source" in the editor and remove the icon tag.
```

Comment:
```suggestion
    If for some reason the content of the plugin is empty, try positioning the cursor behind it empty object and use backspace.
```
That's actually a misconfiguration in the standard setup of the icon plugin. The standard text-editor to be (djangocms-text) does not offer a "source" view. We need to fix the standard config for icons.

Reply:
Is there a reason why it would not offfer "source" view?